src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -44,10 +46,14 @@ def load_arguments(self, _):
         c.argument('admin_enabled', help='Indicates whether the admin user is enabled', choices=['true', 'false'])
 
     with self.argument_context('acr repository delete') as c:
-        c.argument('manifest', nargs='?', required=False, const='', default=None, help='The sha256 based digest of manifest to delete')
+        c.argument('manifest', nargs='?', required=False, const='', default=None, help=argparse.SUPPRESS)
         c.argument('yes', options_list=['--yes', '-y'], action='store_true', help='Do not prompt for confirmation')
-        c.argument('repository', help='The name of the repository to delete.')
-        c.argument('tag', help='The name of tag to delete')
+        c.argument('repository', help="The name of the repository to delete. For example, 'hello-world'")
```

Comment:
No need to include the `For example` clause here.

Reply:
Do you suggest to remove the whole `For example, 'hello-world'` or just `For example`?

Reply:
The whole sentence/clause, please. Applies to all my comments about `For example` text.